content/accessible/control-focus-with-tabindex/index.md

```diff
@@ -47,8 +47,8 @@ Insert an element into the natural tab order using `tabindex="0"`. For example:
 <div tabindex="0">Focus me with the TAB key</div>
 ```
 
-The element can be focused by pressing the `TAB` key and by calling its
-`focus()` method.
+The element comes into focus when the `TAB` key is pressed or when its `focus()`
+method is called.
```

Comment:
@Meggin which of these two treatments do you prefer?

Live example: https://web.dev/accessible/control-focus-with-tabindex#insert-an-element-into-the-tab-order

I was wondering if it should be phrased like, "Press the TAB key to focus the element. Alternatively, call the element's focus() method"

Reply:
Hey, I like the second version-- active voice, 'Press the TAB key...'

Reply:
Updated!